client/my-sites/customer-home/go-mobile-card/style.scss

```diff
@@ -0,0 +1,33 @@
+.go-mobile-card.is-single-store {
+	display: flex;
+	padding-bottom: 1rem;
+}
+
+.go-mobile-card.is-single-store .get-apps__app-badge {
+	width: auto;
+}
+
+.go-mobile-card__app-badges {
```

Comment:
I think these CSS rules are breaking the changes added in #38940 

Before | After
--- | ---
<img src="https://user-images.githubusercontent.com/1233880/72726559-ed3cd780-3b88-11ea-8fcd-67ec61f93e8b.png"> | <img src="https://user-images.githubusercontent.com/1270189/72764523-62b2a300-3b9d-11ea-9374-4dae3cb3c818.png">

Reply:
Maybe they should be defined before `.go-mobile-card.is-single-store`

Reply:
Fixed in #3c48066